code/modules/hydroponics/hydroponics.dm

```diff
@@ -1299,3 +1305,21 @@
 /obj/item/circuit_component/hydroponics/input_received(datum/port/input/port)
 	if(attached_tray.anchored && attached_tray.powered())
 		attached_tray.set_self_sustaining(!!selfsustaining_setting.value)
+
+/particles/pollen
```

Comment:
do we not have a dedicated particles folder? i don't know if that is a better place to put this sort of thing

Reply:
From last I saw, we don't buuuuuuuuut folder are free so I might as well make one